### PR TITLE
perf(app): #33 optimize all images in what we do section

### PR DESCRIPTION
**Pull request by MrGreenTea**
This includes a refactor to astro for the component.

This is important because these are all very high-res images (8MB+) and therefore load slowly on end-user devices and also eat up all our Vercel bandwidth:

image

Related to #33

### CODE REVIEW

**Comment by MrGreenTea** [comment]
@Han2-Ro There is an issue I just noticed: Expanding the exercises does not work anymore. This is not covered by our tests yet!

**Comment by MrGreenTea** [comment]
I'm really confused. It does not work in `dev` but with `pnpm run build && pnpm run preview` it works like a charm.

In dev mode I get this error:
```
Error hydrating /src/components/about-us/Exercises.svelte SyntaxError: The requested module '/node_modules/.pnpm/cssesc@3.0.0/node_modules/cssesc/cssesc.js?v=38871376' does not provide an export named 'default' (at transition.js?v=38871376:3:8)
start @ about-us:2408
await in start (async)
childrenConnectedCallback @ about-us:2408
await in childrenConnectedCallback (async)
connectedCallback @ about-us:2408
m @ @astrojs_svelte_client__js.js?v=38871376:43
mount @ MeltPortalled.svelte:11
mount_component @ chunk-6C6FYBOG.js?v=38871376:2134
init @ chunk-6C6FYBOG.js?v=38871376:2215
MeltPortalled @ MeltPortalled.svelte:6
createProxiedComponent @ svelte-hooks.js?v=38871376:341
ProxyComponent @ proxy.js?v=38871376:242
Proxy<MeltPortalled> @ proxy.js?v=38871376:349
(anonymous) @ @astrojs_svelte_client__js.js?v=38871376:19
(anonymous) @ about-us:2408
e @ about-us:2408
await in e (async)
start @ about-us:2408
childrenConnectedCallback @ about-us:2408
await in childrenConnectedCallback (async)
e @ about-us:2408
(anonymous) @ about-us:2408
Show 7 more frames
Show less
```

**Comment by MrGreenTea** [comment]
What's so weird about it is that the supporting member component in /support-us works fine and does the same thing. Both use `melt-ui` and a `transition:slide` ...

**Comment by MrGreenTea** [comment]
Very very weird: It suddenly stopped happening and now works reliably??

File: tests/about-us.spec.ts [hunk 1]
```diff
@@ -1,14 +1,30 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, type Page } from '@playwright/test';
 
 test.beforeEach(async ({ page }) => {
 	await page.goto('/about-us');
 });
 
+async function forceLoadImages(page: Page) {
```
Comment (Han2-Ro) [inline tests/about-us.spec.ts:7]: This function is pretty cool. Probably useful for the other tests, too.

**Review by Han2-Ro — APPROVED** [review]
weird stuff with the exercises, but if works now it works. So  lgtm

File: tests/about-us.spec.ts (hunk 1 above)
Comment (MrGreenTea) [reply tests/about-us.spec.ts:7]: @Han2-Ro I think so too. Maybe create a follow-up issue to clean up the tests and make them use this function?

**Pull request merged**